assertions/confirmation.go

```diff
@@ -92,6 +92,10 @@ func (m *Manager) updateLatestConfirmedMetrics(ctx context.Context) {
 				srvlog.Debug("Could not fetch latest confirmed assertion", log.Ctx{"error": err})
 				continue
 			}
+			if _, ok := m.assertionChainData.canonicalAssertions[latestConfirmed.Id()]; !ok {
+				srvlog.Debug("Latest confirmed assertion not in canonical mapping", log.Ctx{"assertionHash": latestConfirmed.Id().Hash})
```

Comment:
This should be a warn and be more serious, saying that "Evil assertion was possibly confirmed"

Reply:
done.